Tests/MongoSwiftTests/CommandMonitoringTests.swift

```diff
@@ -34,20 +32,15 @@ final class CommandMonitoringTests: XCTestCase {
 
             let testFilePath = URL(fileURLWithPath: "\(cmPath)/\(filename)")
             let asDocument = try Document(fromJSONFile: testFilePath)
-            let testFile = try CMTestFile(fromDocument: asDocument)
+            let testFile = try decoder.decode(CMTestFile.self, from: asDocument)
 
             print("-----------------------")
             print("Executing tests for file \(name)...\n")
 
             // execute the tests for this file
             for var test in testFile.tests {
 
-                // Check that the current version we're testing is within bounds for this cases
-                if let maxVersion = test.maxServerVersion, maxVersion < VERSION {
-                    continue
-                } else if let minVersion = test.minServerVersion, VERSION < minVersion {
-                    continue
-                }
+                if try !client.serverVersionIsInRange(test.minServerVersion, test.maxServerVersion) { continue }
```

Comment:
prefer some sort of output indicating that a test was skipped rather than silently continuing.

Reply:
I think I already asked this, but XCTest has no notion of skipped tests, correct?

Is more custom output (thinking of the noisy CRUD tests) actually useful, given that we really have no way to meaningfully interpret it during CI? Perhaps this is something to put off until we can adopt a more full-featured test framework.

Reply:
yeah, AFAIK there is no easy way to indicate a test should be skipped in XCTest. 

I agree that output is a good idea, though yes, it would be nice if it were more meaningful in CI. unfortunately testing from the command ljne in Swift is kind of garbage... when we explored other test frameworks like Quick a while back we ended up just coming back to XCTest, lol.